boilerplate/README.md

```diff
@@ -1,67 +1,137 @@
 #  <%= props.name %>
-[![js-standard-style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg?style=flat)](http://standardjs.com/)
 
-* Standard compliant React Native App Utilizing [Ignite](https://github.com/infinitered/ignite)
+[![Build Status](https://semaphoreci.com/api/v1/ir/ignite-ir-boilerplate-bowser/branches/master/badge.svg)](https://semaphoreci.com/ir/ignite-ir-boilerplate-bowser)
 
-## :arrow_up: How to Setup
 
-**Step 1:** git clone this repo:
 
-**Step 2:** cd to the cloned repo:
+## The latest and greatest boilerplate for Infinite Red opinions
 
-**Step 3:** Install the Application with `yarn` or `npm i`
+This is the boilerplate that [Infinite Red](https://infinite.red) uses as a way to test bleeding-edge changes to our React Native stack.
 
+Currently includes:
 
-## :arrow_forward: How to Run App
+* React Native 0.55.1 (but you can change this if you want to experiment)
```

Comment:
We actually have 0.55.4
https://github.com/infinitered/ignite-ir-boilerplate-bowser/blob/master/lib/react-native-version.js#L4

You sure you want to include this text?  Might be something we have to keep updating all the time...